packages/react-components/react-jsx-runtime/src/createElement.ts

```diff
@@ -10,40 +10,29 @@ export function createElement<P extends {}>(
   props?: P | null,
   ...children: React.ReactNode[]
 ): React.ReactElement<P> | null {
-  if (!isSlotComponent(props)) {
-    return React.createElement(type, props, ...children);
+  return isSlotComponent(props)
```

Comment:
Let's also rename it to something like `isSlotThatContainsRenderFunction` or `hasRenderFunction`.

Reply:
Sounds good to me, I guess I kept the name as `isSlotComponent` thinking on the future implementation